### ISSUE TITLE
Details about training corpus, availability of pre-trained model and data

### ISSUE THREAD

**Issue by gurunathparasaram** [zhawe01/fairseq-gec#1]
Thanks for open sourcing the code. I have a few queries:
* For pretraining and adding noise, 'train_1b.tgt' is used. Is this the same as concatenating all the files from ![Google 1B dataset](http://www.statmt.org/lm-benchmark/1-billion-word-language-modeling-benchmark-r13output.tar.gz). Was any further pre-processing done?
* Can a link to the pre-processed data and pre-trained files be provided?
Thanks!

**Comment by zhawe01** [comment]
- The "train_1b.tgt" file is built from the One Billion Benchmark. Tokenization was applied to each sentence.

- I will share them later.

**Comment by gurunathparasaram** [comment]
Thank you

**Comment by gurunathparasaram** [comment]
Can you please share the pre-processed data and pre-trained models?

**Comment by gurunathparasaram** [comment]
Can you share the ablation metrics for the pre-trained model with and without using a spell-corrector? Thanks.

**Comment by zhawe01** [comment]
I didn't do that, but later I can provide the spelling error corrected source sentences of the CoNLL 2014 test dataset.

**Comment by gurunathparasaram** [comment]
Thanks.

**Comment by gurunathparasaram** [comment]
Can you please provide the spell-corrected source sentences on CoNLL-14 test set?
Thanks.

**Comment by zhawe01** [comment]
Already shared.

**Comment by luopeixiang** [comment]
Can you please provide the link to download train_1b.tgt?

**Comment by zhawe01** [comment]
> Can you please provide the link to download train_1b.tgt?

It is the tokenized One Billion Benchmark dataset. You can download and tokenize it yourself.

**Comment by bozliu** [comment]
Could you tell me where I can find the file or directory: './data_art/train_1b_1.src'.
